tests/describe_task_queue.go

```diff
@@ -122,42 +130,105 @@ func (s *DescribeTaskQueueSuite) publishConsumeWorkflowTasksValidateBacklogInfo(
 
 	// call describeTaskQueue to verify if the backlog decreased
 	expectedBacklogCount[enumspb.TASK_QUEUE_TYPE_WORKFLOW] = int64(0)
-	s.validateDescribeTaskQueue(tl, expectedBacklogCount)
+	s.validateDescribeTaskQueue(tl, expectedBacklogCount, isEnhancedMode, true, true, workflows)
+}
+
+func (s *DescribeTaskQueueSuite) validateBacklogHeadCreateTime(queueType enumspb.TaskQueueType,
```

Comment:
can you name this something other than `validate...`? that implies that it should be asserting something (fail the test if it's false). i.e. it's confusing that validateDescribeTaskQueue has side effects and returns nothing but validateBacklogHeadCreateTime has no side effects and returns a bool.

maybe isBacklogHeadCreateTimeCorrect or something like that?